RetrospectiveExtension.Frontend/scripts/ci.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+##
+## This script runs the CI tasks
+##
+(
+    cd "$(dirname "$0")/.." || exit
+    set -euo pipefail
+
+    npm install
+    echo "install dependencies"
+
+    npm run lint
+    echo "eslint passed"
+
+    npm run build:p
+    echo "build passed"
+
+    npm run test
+    echo "tests passed"
+
```

Comment:
Did we forget adding the `npm run pack:p` command? This helps validate the extension json file.

Reply:
I can add that back. I was focusing on CI but you make a good point about validating the manifest.

Reply:
I ran into an interesting problem when adding pack:p. Since the pre-commit operation stashes unstaged changes while running hooks the update of the manifest (version increment) is conflicting with the stash apply and the commit is rolled back. The error message is "[WARNING] Stashed changes conflicted with hook auto-fixes... Rolling back fixes...". To work around this I added a new pack:test script that does not include the --rev-version flag. This will validate the manifest and allow the pre-commit hook to operate as intended.

Reply:
Make senses, thanks for explaining.